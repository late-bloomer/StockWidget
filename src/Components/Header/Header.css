.leftText {
  color: white;
  font-size: 24px;
  flex: 35%;
  padding: 24px;
}
.widText {
  color: white;
  font-size: 24px;
  flex: 65%;
  padding: 24px 12px 24px 32px;
}
.headerContainer {
  display: flex;
  height: 82px;
  background: blue;
  box-sizing: border-box;
}

@media screen and (max-width: 500px) {
  .leftText {
    font-size: 16px;
  }
  .widText {
    font-size: 16px;
    padding: 24px 12px 24px 0;
  }
}
