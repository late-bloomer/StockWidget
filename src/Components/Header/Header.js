import React from "react";
import "./Header.css";
function Header() {
  return (
    <div className="headerContainer">
      <div className="leftText">Hello Intuit</div>
      <div className="widText">STOCK PICKER WIDGET</div>
    </div>
  );
}

export default Header;
