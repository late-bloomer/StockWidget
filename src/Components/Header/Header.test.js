import React from "react";
import ReactDOM from "react-dom";
import Enzyme, { shallow, render, mount } from "enzyme";
import Header from "./Header";
import toJson from "enzyme-to-json";
import Adapter from "enzyme-adapter-react-16";

Enzyme.configure({ adapter: new Adapter() });

describe("Header Component", () => {
  it("should render Header Successfully", () => {
    const wrapper = shallow(<Header />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
