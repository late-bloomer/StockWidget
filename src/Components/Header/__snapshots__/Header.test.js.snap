// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header Component should render Header Successfully 1`] = `
<div
  className="headerContainer"
>
  <div
    className="leftText"
  >
    Hello Intuit
  </div>
  <div
    className="widText"
  >
    STOCK PICKER WIDGET
  </div>
</div>
`;
